docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -0,0 +1,783 @@
+
+  ```code
+  $ git clone https://github.com/gravitational/teleport-plugins.git --depth 1
+  $ cd teleport-plugins/event-handler/build.assets
+  $ make build
+  ```
+
+  You can find the compiled binary within your clone of the `teleport-plugins`
+  repo, with the file path, `event-handler/build/teleport-event-handler`. Move
+  this into your `PATH`.
+
+  </TabItem>
+  <TabItem options="Executable" label="From Source ">
+
+  You will need Go >= (=teleport.golang=) installed.
+
+  Run the following commands on your Universal Forwarder host:
+
+  ```code 
+  $ git clone https://github.com/gravitational/teleport-plugins.git --depth 1
+  $ cd teleport-plugins/event-handler
+  $ go build
+  ```
+
+  The resulting executable will have the name `event-handler`. To follow the
+  rest of this guide, rename this file to `teleport-event-handler` and move it
+  into your `PATH`.
+
+  </TabItem>
+</Tabs>
+
+### Generate a starter config file
+
+Generate a configuration file with placeholder values for the Teleport Event
+Handler plugin. Later in this guide, we will edit the configuration file for
+your environment.
+
+On the host where you are running the Universal Forwarder and the Event Handler
+plugin, execute the following command to generate a sample configuration that we
+will edit later. Replace `mytenant.teleport.sh` with the DNS name of your
+Teleport Proxy Service or Teleport Cloud tenant:
+
+```code
+$ teleport-event-handler configure . mytenant.teleport.sh
+```
+
+The `teleport-event-handler configure` command generates several files that you
+will refer to when generating credentials for the Event Handler plugin and
+configuring the Universal Forwarder:
+
+|File(s)|Purpose|
+|---|---|
+| `ca.crt` | Self-signed CA certificate that the Event Handler plugin and the Universal Forwarder will use to validate one another's TLS certificates.|
+| `server.crt` and `server.key`| TLS certificate and key for the Universal Forwarder.|
+| `client.crt` and `client.key`| TLS certificate and key for the Event Handler plugin.|
+| `teleport-event-handler-role.yaml`| `user` and `role` resource definitions for the Event Handler plugin.|
+| `fluent.conf`| Fluentd configuration file. We will use parts of this to configure the Universal Forwarder.|
+| `teleport-event-handler.toml` | Starter configuration file for the Event Handler plugin.|
+| `ca.key` | Not used in this setup. |
+
+### Define RBAC resources
+
+The `teleport-event-handler configure` command generated a file called
+`teleport-event-handler-role.yaml`. This file defines a `teleport-event-handler`
+role and a user with read-only access to the `event` API:
+
+```yaml
+kind: role
+metadata:
+  name: teleport-event-handler
+spec:
+  allow:
+    rules:
+      - resources: ['event', 'session']
+        verbs: ['list','read']
+version: v5
+---
+kind: user
+metadata:
+  name: teleport-event-handler
+spec:
+  roles: ['teleport-event-handler']
+version: v2
```

Comment:
Does it matter that there's a version mismatch between these two entities?

Reply:
Configuration resources are versioned separately, e.g., here are all the versions used in the `master` branch of `gravitational/teleport`:

```
$  grep -REo "type [A-Z][A-Za-z]+SpecV[0-9]+ struct" ./api/types | grep -Eo "V[0-9]+" | sort | uniq -c
   4 V1
  16 V2
  17 V3
   1 V4
   1 V5
```